### PR TITLE
Feature/iii 3588 loading spinner component

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- Spinner component
- Spinner story

---

Ticket: https://jira.uitdatabank.be/browse/III-3588

### CODE REVIEW

File: src/components/publiq-ui/Spinner.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { Spinner as BootstrapSpinner } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const SpinnerVariants = {
+  PRIMARY: 'primary',
+  LIGHT: 'light',
+};
+
+const SpinnerSizes = {
+  NORMAL: undefined,
```
Comment (brampauwelyn) [inline src/components/publiq-ui/Spinner.js:11]: undefined will result in the default spinner size?

**Review by brampauwelyn — APPROVED** [review]

File: src/components/publiq-ui/Spinner.js (hunk 1 above)
Comment (simon-debruijn) [reply src/components/publiq-ui/Spinner.js:11]: jep

Comment (geoffreydhuyvetters) [reply src/components/publiq-ui/Spinner.js:11]: not completely clear why we're doing this? we have a defaultProp right?

Comment (simon-debruijn) [reply src/components/publiq-ui/Spinner.js]: I've fixed it, better?

**Pull request merged**